style.css

```
@@ -1,11 +1,41 @@
+body {
+  margin: 0;
+}
+
+/* PC ver*/
+@media (min-width: 1200px) {
+  .container {
+    width:50%;
+
+  }
+/*  Tablet ver */
+@media (max-width: 1199px) and (min-width: 768px) {
+  body {
+    width: 70%;
+  }
+  .header {
+    padding: 0 24px;
+  }
+}
+/*  Mobile ver */
+@media (max-width: 767px) and (max-width: 375px) {
+  body {
+    width:90%;
+  }
+  .header {
+    padding: 0 16px;
+  }
+}
+
+}
```

❗️ 수정요청
PC 미디어 쿼리문안에서 중첩될 수 없는 조건문안에 tablet 과 moblie 이 작성되어 있어 적용이 되지 않네요!
밖으로 빼주세요~

```suggestion
/* PC ver*/
@media (min-width: 1200px) {
  .container {
    width: 50%;
  }
}

/*  Tablet ver */
@media (max-width: 1199px) and (min-width: 768px) {
  body {
    background-color: red;
    width: 70%;
  }
  .header {
    padding: 0 24px;
  }
}

/*  Mobile ver */
@media (max-width: 767px) and (max-width: 375px) {
  body {
    width: 90%;
  }
  .header {
    padding: 0 16px;
  }
}
```